config/custom-environment-variables.yaml

```diff
@@ -30,4 +30,10 @@ policy:
     inCluster:
       __name: KUBEMAIL_KUBERNETES_INCLUSTER
       __format: json
-    static: KUBEMAIL_KUBERNETES_POLICY
\ No newline at end of file
+    static: KUBEMAIL_KUBERNETES_POLICY
+
+watcher:
+  emailPolicyInformer:
+    selector: WATCHER_EMAILPOLICYINFORMER_SELECTOR
```

Comment:
What should be the content of this environment variable? Seems like `watcher.{emailPolicyInformer,podInformer}.selector` are both expecting object values. In this case, this should maybe have a `__format: json` property.